AdobeStockImage/Model/GetImageList.php

```diff
@@ -40,42 +40,73 @@ class GetImageList implements GetImageListInterface
      */
     private $filterBuilder;
 
+    /**
+     * @var string $defaultGalleryId
+     */
+    private $defaultGalleryId;
+
     /**
      * GetImageList constructor.
      *
      * @param FilterGroupBuilder $filterGroupBuilder
      * @param GetAssetListInterface $getAssetList
      * @param FilterBuilder $filterBuilder
      * @param array $defaultFilters
+     * @param string $defaultGalleryId
      */
     public function __construct(
         FilterGroupBuilder $filterGroupBuilder,
         GetAssetListInterface $getAssetList,
         FilterBuilder $filterBuilder,
-        array $defaultFilters = []
+        array $defaultFilters = [],
+        string $defaultGalleryId
     ) {
         $this->getAssetList = $getAssetList;
         $this->filterGroupBuilder = $filterGroupBuilder;
         $this->defaultFilters = $defaultFilters;
         $this->filterBuilder = $filterBuilder;
+        $this->defaultGalleryId = $defaultGalleryId;
     }
 
     /**
      * @inheritdoc
      */
     public function execute(SearchCriteriaInterface $searchCriteria): SearchResultInterface
     {
+        $searchCriteria = $this->setCuratedFilter($searchCriteria);
         $searchCriteria = $this->setDefaultFilters($searchCriteria);
+
         return $this->getAssetList->execute($searchCriteria);
     }
 
+    /**
+     * Set Curated filter if not set other filters.
+     *
+     * @param SearchCriteriaInterface $searchCriteria
+     * @return SearchCriteriaInterface
+     */
+    private function setCuratedFilter(SearchCriteriaInterface $searchCriteria): SearchCriteriaInterface
```

Comment:
It's better to rename this method to match technical naming
```suggestion
    private function setDefaultGallery(SearchCriteriaInterface $searchCriteria): SearchCriteriaInterface
```

Reply:
Ok